code/game/atoms.dm

```diff
@@ -634,4 +634,16 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 	return
 
 /atom/proc/wirecutter_act(mob/user, obj/item/tool)
-	return
\ No newline at end of file
+	return
+
+/atom/proc/on_brew()
+	return
+
+/atom/proc/on_smelt()
+	return

```

Comment:
the idea of every single atom having those procs is both amusing and terrifying to me.

Reply:
goof code